pyproject.toml

```diff
@@ -52,69 +53,59 @@ version = "0.1.1"
 [tool.poetry.dependencies]
 bidict = "^0.23.1"
 numba = [
-  {python = ">=3.9,<4.0", version = "^0.60.0"},
-  {python = "3.8", version = "^0.58.0"}
+  {python = ">=3.9,<3.13", version = "0.60.0"},
+  {python = ">=3.13,<3.14", version = "0.61.0"}
 ]
 numpy = [
-  {python = "^3.9", version = "^1.26"},
-  {python = "^3.8, <3.12", version = "^1.24"}
+  {python = ">=3.9,<3.13", version = ">=1.24,<2.0"},
+  {python = ">=3.13,<3.14", version = ">=2.1,<3.0"}
```

Comment:
Please use the caret symbol instead of multiple greater than / less than symbols